RegionTypes/aurabar.lua

```diff
@@ -1152,21 +1159,15 @@ local function modify(parent, region, data)
         if customValue then
       -- Update via custom OnUpdate handler
             if type(customValue) == "function" then
-                local value, total = customValue(data.trigger);
+                local value, total = customValue(region.state.trigger);
```

Comment:
It's no longer always the first trigger that is active, data.trigger is no longer correct.